step.sh

```diff
@@ -29,6 +29,7 @@ upload_app() {
 
   curl -# -X POST \
     -H "Authorization: token $api_key" \
+    -A "bitrise/deploygate--upload-app-bitrise-step" \
```

Comment:
We should follow the RFC's UserAgent format.
http://www.ietf.org/rfc/rfc2616.txt

the format: `User-Agent     = "User-Agent" ":" 1*( product | comment )`
the format of `product` : `<product name>[/<product version]` 
an example: `User-Agent: CERN-LineMode/2.15 libwww/2.17b3`

So `DeployGateUploadApp-BitriseStep/<version>` would be better. Also, we need to inject `version` when creating a release artifact like below.

```
#bin/bash

STEP_VERSION=1.0.0

...
  -A "DeployGateUploadApp-BitriseStep/$STEP_VERSION" \
...
```

Reply:
thanks! i will change it.